tests/containers/container_engine.pm

```diff
@@ -128,7 +128,7 @@ sub basic_container_tests {
     validate_script_output("$runtime image ls", qr/tumbleweed/, fail_message => "Tumbleweed image removed, despite being in use");
     assert_script_run("$runtime system prune -f");
     validate_script_output("$runtime image ls", qr/tumbleweed/, fail_message => "Tumbleweed image removed, despite being in use");
-    assert_script_run("! $runtime rmi -a");    # should not be possible because image is in use
+    assert_script_run("! $runtime rmi $image");    # should not be possible because image is in use
```

Comment:
Would be better as if forces all images to be delete `$runtime image rm -f $($runtime images -q)`